### ISSUE TITLE
no damage on save seems bugged

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#559]
In GitLab by @siliconsaint on Oct 7, 2021, 23:42

I have the feature checked to look for "half as much" damage, but regardless it seems to be applying half damage on save.

![Screen_Shot_2021-10-07_at_8.36.50_AM](https://my-gitlab-bucket.s3.amazonaws.com/640711674/66c910d88ea5c2a79972f9a5a3884c214f6309cc557297270d51bfd8e1475422/Screen_Shot_2021-10-07_at_8.36.50_AM.png)

Note the description does not include "half as much" so it should waive damage on save?

Foundry 8.9
DND5e 1.43
Midi-qol 8.68

Find attached midi-qol config along with spell card.
[fvtt-midi-qol-settings__9_.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/87fdaa821d4a2bd90897d644bc46193d33af04a64a34f83fc4fc55413223ad97/fvtt-midi-qol-settings__9_.json)[fvtt-Item-hunger-of-hadar-_ends-turn_.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/05fdcc4bb46828a7f5b1c50c5086efa1587b8c3da23421b82da4e8bc463db90d/fvtt-Item-hunger-of-hadar-_ends-turn_.json)

**Comment by tposney** [comment]
Apologies this is not well explained in the readme (I did try).

If you want a spell to do no damage on save, midi ALWAYS searches for "no damage on save" in the spell description and if found saves will do no damage. So that is one way and always works whatever the settings.

The other way relies on the settings.
There are 2 relevant settings search spell text and default save multiplier.
![Screenshot_from_2021-10-08_12-39-00](https://my-gitlab-bucket.s3.amazonaws.com/640711674/01d14d53b93ce822477e35455dee26b13e77a3cdf8dbeaad744ab356632f0254/Screenshot_from_2021-10-08_12-39-00.png)

If the "search spell text is disabled" midi will use the default save multiplier for spells that save that did not match "no damage on save".

If search spell text is enabled then spells that have the "half damage on save" text will do 1/2 damage on a save and spells that do not have the text will do default saving throw multiplier on save.

So in summary if you want the default to be no damage on save change the default multiplier to 0 and enable search spell text.
If you want spells to do full damage on save (effectively the save only influences active effects that are applied) set the default multiplier to 1.0.

**Comment by tposney** [comment]
In GitLab by @siliconsaint on Oct 8, 2021, 13:10

Tim, thanks for the comprehensive response, I get it now!!
